tests/pytorch/test_hook_paths.py

```diff
@@ -0,0 +1,45 @@
+# Standard Library
```

Comment:
related to this PR?

Reply:
related because of CI changes. this test was in tests/core before. with the new CI, this test wont work in a container other than pytorch. similarly test_hook_collections.py wont work in any container other than mxnet. we could either move it to the respective framework directory like this or ignore in tests.sh,  https://github.com/awslabs/sagemaker-debugger/blob/d446b26ca11f3ff655f24441af749a26eab560d1/config/tests.sh#L20.

Note: if we are making this change of moving to individual directory, then tests.sh has to be updated. will do this when i push the next commit.